pypeit/core/wavecal/autoid.py

```diff
@@ -854,8 +854,46 @@ def reidentify(spec, spec_arxiv_in, wave_soln_arxiv_in, line_list, nreid_min, de
     return detections, spec_cont_sub, patt_dict_slit
 
 
+def measure_fwhm(spec, binspectral):
```

Comment:
Thanks for pulling this out.  Next question -- can we call it from the other methods (at least `reidentify`) so that we have `measured_fwhm` for those spectrographs (mainly the echellettes)?

Reply:
I have added `measured_fwhm` for the `reidentify` method.  The use of `measured_fwhm` is contingent to having the parameter `fwhm_fromlines` set to `True`, which is currently the default only for DEIMOS and MOSFIRE.

Reply:
A couple lingering questions:

- This function no longer requires `binspectral` as an argument, right?
- I just want to confirm that you've checked a case when the binning of the arcs and science frames are different.  Is everything propagating as it used to?

Reply:
@kbwestfall, for your first point, your right! I forgot that. Thanks. I'll fix it.

For the second point, the part of the code that I'm touching is only related to the FWHM that is used to identify arc lines. In this case, I'm pretty sure the arc spectrum is used as it is (i.e., with whatever binning it has), therefore the FWHM should also be expressed in binned pixels of the arc. After this step, I'm not really sure how the code handles cases when the binning of the arcs and science frames are different, but it should not affected by the changes I made here.